### ISSUE TITLE
Slime Transform Markings

### ISSUE THREAD

**Issue by TripleShades** [citadel-station/citadel-main#15087]
Issue reported from  Round ID: 28265 (Citadel Station)

Reporting client version: 514.1556

Slime transform markings system is nonfunctional and does not apply any markings, possibly broken by #14653

**Comment by BlueWildrose** [comment]
Yeah that hasn't been updated at all really, the slime transformation is bugged in so many regards. Hopefully with what I got planned that it'll all eventually be fixed.

It's still in the prototype stages though so maybe if someone can whip together a fix for the current slime shapeshifting issues that'd be nice.

**Comment by buffyuwu** [comment]
@timothyteakettle

**Comment by timothyteakettle** [comment]
hi i'll look into a quickfix for this probably sometime on monday, however due to individual markings its a bit of a pain
